settings.gradle

```diff
@@ -14,4 +14,4 @@
  * limitations under the License.
  */
 
-include "nanofix-client", "integration-tests"
\ No newline at end of file
+include "nanofix-client" , "integration-tests"
```

Comment:
Not entirely sure this space needs to be here

Reply:
Fixed.